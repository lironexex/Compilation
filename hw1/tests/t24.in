// comments 4 - empty comments
// the empty single-line comment in the next line is legal:
//
//the next empty multi-line comment is also legal:
/**/
