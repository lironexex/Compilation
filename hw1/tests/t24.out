1 COMMENT 1
2 COMMENT 1
3 COMMENT 1
4 COMMENT 1
5 COMMENT 1
